src/components/AddResourceModal/AddResourceModalV2.jsx

```diff
@@ -237,80 +190,147 @@ export default function AddResourceModalV2(props) {
       .catch(console.error);
   };
 
-  const onSubmit = e => {
+  const onSubmit = (resourceType, e) => {
```

Comment:
if we only pass the event for preventing default, let's prevent default before calling this function.